tests/test_gaussian_gjf.py

```diff
@@ -0,0 +1,16 @@
+import os
+import unittest
+from context import dpdata
+
+
+class TestDumpGaussianGjf(unittest.TestCase):
+    def setUp(self):
+        self.system = dpdata.LabeledSystem('gaussian/methane.gaussianlog', 
+                                           fmt = 'gaussian/log')
+    
+    def test_dump_to_gjf(self):
+        self.system.to_gaussian_gjf("gaussian/tmp.gjf")
```

Comment:
Do you have an expected content of the output file?

What is written in `gaussian/tmp.gjf` can be compared to the reference.

Reply:
Of course. But I'm not sure if it's necessary to compare them in a test case, for that the best way to check an input file is to run it with Gaussian.

Reply:
It is necessary to ensure what the method generated is what you want.

Reply:
OK. I've added test case for that function.